data/description_text.yaml

```diff
@@ -18,19 +18,11 @@ description_text-markdown: |+
 directions_text-markdown: |+
   ## How to use the map:
   
-  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod 
-  tempor incididunt ut labore et dolore magna aliqua. Consequat ac felis
-  donec et odio pellentesque. Sapien faucibus et molestie ac feugiat. 
-  Morbi tristique senectus et netus et malesuada fames ac. Tempus urna 
-  et pharetra pharetra massa massa ultricies mi quis. Risus nec feugiat 
-  in fermentum. Condimentum mattis pellentesque id nibh tortor id aliquet. 
-  Tortor dignissim convallis aenean et tortor at risus. Praesent elementum
-  facilisis leo vel fringilla. Adipiscing tristique risus nec feugiat in 
-  fermentum. Lacus laoreet non curabitur gravida arcu ac. Rutrum tellus 
-  pellentesque eu tincidunt tortor aliquam nulla. Massa eget egestas purus 
-  viverra accumsan in nisl nisi scelerisque. Faucibus turpis in eu mi 
-  bibendum neque egestas congue quisque. Purus sit amet luctus venenatis 
-  lectus magna fringilla urna. Odio ut enim blandit volutpat maecenas 
-  volutpat. Aliquam sem et tortor consequat id porta. Nunc lobortis mattis 
-  aliquam faucibus. Odio pellentesque diam volutpat commodo sed. Arcu dictum
-  varius duis at consectetur lorem donec massa sapien.
+  Explore how water is used throughout the United States. Click or 
+  hover over on any of the specific use-case boxes below the map to 
+  discover how thermoelectric, public supply, irrigation, or industrial 
+  water use is distributed. Click on a state to zoom in to more easily see the
+  difference in county use. Once zoomed in, simply click anywhere outside
+  the state on the map to return to the full country view. Hover (tap on 
+  mobile) over any of the individual counties to get more information.
```

Comment:
Currently, you can only hover over the circle for each county, not the county itself.

Reply:
True...but is that how we want it to be for all time? I've been wondering whether we should try to make the counties clickable as well.

Reply:
Yes...but what if we don't end up doing it and forget to update the text?

Reply:
It would be great to be able to hover over the county that has the dot and get the tip for it. I assume this is a target for us, unless the county geoms are limiting performance in a major way. We also might want to get further down the path of what we want the end result to be before spending too much time altering the text. For example - does clicking a neighboring state once zoomed in pan you to that state-zoom, or zoom you out to national? I don't know that we've resolved some of these questions (or started the discussions).

Reply:
Yeah, fair. Maybe there's no way to predict right now what the text will need to be. I'm ok with leaving the text as-is and remembering to refine the details once we've finalized the behavior.

Performance aside, is it going to be easy to get the hover behavior to be intuitive when it includes both counties and dots? big dots extend into other counties...when you hover over County A's dot where it extends into County B, which hover text should come up, and can we easily implement it that way?

I'm not sure how many of these design decisions need to turn into full-on discussions. There are many such decisions, and open-ended discussions are costly. I like the current zoom behavior, which is why I wrote it that way, though it's also absolutely possible to pan to a neighboring state. Jordan or others, if you have specific views and want to question the current behavior, could you create an issue proposing the approach you'd prefer? Then if anybody disagrees, it can be up to them to make a counter proposal within that issue, and if it becomes clear that the topic needs live discussion, we can plan to have that discussion during a team meeting.

Reply:
Gotcha. The question about click/zoom is in here: https://github.com/USGS-VIZLAB/water-use-15/issues/63 `Do we want to mimic the behavior (zoom in/out, and click to adjacent states) from here: https://projects.fivethirtyeight.com/facebook-primary/ ?`

Reply:
oh, yep, there's a lot there